### PR TITLE
[WSTEAM1-96] Instagram & Youtube Social Embed (TEST)

### PR DESCRIPTION
**Pull request by mfonofm**
Resolves [WSTEAM1-96](https://jira.dev.bbc.co.uk/browse/WSTEAM1-96)

**Overall change:**
Enable Instagram and Youtube embeds on Optimo Test environment

**Code changes:**
- Adds a new page handler that adds -nocookie to youtube social links
- Enables Instagram and Youtube embeds by inferring the correct provider
 

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/legacy/containers/SocialEmbed/index.jsx [hunk 1]
```diff
@@ -13,22 +13,34 @@ import { GridItemMedium } from '#components/Grid';
...
+import isLive from '#lib/utilities/isLive';
 import createTranslations from './common/translations';
 import { LAZYLOAD_OFFSET, Wrapper } from './common/styles';
 import { getProviderFromSource, getIdFromSource } from './sourceHelpers';
 
 const logger = nodeLogger(__filename);
 
-const SocialEmbedContainer = ({ blocks, source }) => {
+const SocialEmbedContainer = ({ blocks, source, pageType }) => {
   const { isAmp } = useContext(RequestContext);
```
Comment (RetiredBlob) [inline src/app/legacy/containers/SocialEmbed/index.jsx]: `pageType` is available here - doesn't need to be a new prop, or a new string:
```suggestion
  const { isAmp, pageType } = useContext(RequestContext);
```
Can be compared to the string in `import { ARTICLE_PAGE } from '#app/routes/utils/pageTypes';`

**Review by RetiredBlob — DISMISSED** [review]
👍

File: src/app/legacy/containers/SocialEmbed/index.jsx [hunk 2]
```diff
@@ -13,22 +13,34 @@ import { GridItemMedium } from '#components/Grid';
...
   const { service, translations } = useContext(ServiceContext);
 
   if (!blocks || !source) return null;
-
+  const { model } = blocks[0];
+  console.log(source);
```
Comment (amoore108) [inline src/app/legacy/containers/SocialEmbed/index.jsx]: Couple of `console.log` here, but looks good 👍🏼

**Review by amoore108 — DISMISSED** [review]

**Review by RetiredBlob — APPROVED** [review]
👍

**Review by amoore108 — APPROVED** [review]

**Comment by greenc05** [comment]
A11y checks required on a page with 1 embed and also a page with more than 1 embed of the same type (which I believe you have alraady done)
 
- AAC on ticket https://jira.dev.bbc.co.uk/browse/WSTEAM1-96
- Checking skip link with a keyboard
- Checking skip link with a screen reader
- Check numbering is displayed in both the skip message, url and end message (Only for page with more than 1 embed of the same type)

**Comment by DarioR01** [comment]
> * eck numbering is displayed in both the skip

Double checked, Skip links are 👍 
Screen readers VoiceOver Mac 👍 
Will check with NVDA and TalkBack once on test, currently unable with localhost.

**Pull request merged**